include/fizzy/fizzy.h

```diff
@@ -0,0 +1,18 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+bool fizzy_validate(const uint8_t* wasm_binary, size_t wasm_binary_size);
```

Comment:
What C std are we targeting? C99? EVMC supports C99+C11. We should probably also have a ci run for that.